CITATION.cff

```diff
@@ -0,0 +1,22 @@
+cff-version: 1.2.0
+message: "Please cite the following works when using this software."
+type: software
+title: "vector"
+abstract: "Vector is a Python library for 2D, 3D, and Lorentz vectors, especially arrays of vectors, to solve common physics problems in a NumPy-like way."
+authors:
+- family-names: "Schreiner"
+  given-names: "Henry"
+  orcid: "https://orcid.org/0000-0002-7833-783X"
+  affiliation: "Princeton University"
+- family-names: "Pivarski"
+  given-names: "Jim"
+  orcid: "https://orcid.org/0000-0002-6649-343X"
+  affiliation: "Princeton University"
```

Comment:
@henryiii @jpivarski please check your author information and let me know if there should be other authors added.

Reply:
Given how much he's added to it, I think @Saransh-cpp (Saransh Chopra) should definitely be an author. I don't think he has an [ORCID](https://orcid.org/) yet.

Saranch: It's easy to get one. This is a unique ID that follows all of your academic publications (including code) to disambiguate papers written by people with the same names.

Reply:
These sum up the history rather well (committer # 2 is pre-commit-ci[bot]).

![image](https://user-images.githubusercontent.com/1852447/186511960-1aa5a06c-215a-44c9-b842-ce83d3074ebc.png)

![image](https://user-images.githubusercontent.com/1852447/186512045-b7231b07-ae9b-46e9-9b40-4eb66c6b92e9.png)

![image](https://user-images.githubusercontent.com/1852447/186512086-5b1b3291-a4d8-4c85-ba74-cfed4d10176a.png)

Reply:
Thanks, @jpivarski! I haven't used ORCID ID much, but I believe I had one. I will add it below!